lib/src/util.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:convert';
 
+import '../assets/case_folding.dart';
```

Comment:
Perhaps `src/assets/...` – all things that should not be imported by users should be in `src`

Reply:
Fixed!

Reply:
👍